backend/cms/models/page.py

```diff
@@ -25,6 +25,7 @@ class Page(MPTTModel):
     icon = models.ImageField(blank=True,
                              null=True,
                              upload_to='pages/%Y/%m/%d')
+    order = models.IntegerField(null=True)
```

Comment:
I'm not sure about that, but it could be possible, that the MPTTModel already saves information about the order of nodes.

Reply:
You're right! If I understood it right, we could use the fields `lft` or `rgt` of the MPTT model for ordering, as they should both represent the horizontal ordering. @sascha11110 can you confirm this?

Reply:
Actually, it seems as we don't need to access these order fields directly (it would probably be a bad idea to mess with the tree database manually anyway), we can just use the `move_to(target, position)` method to change the order.